app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchInit.kt

```diff
@@ -7,6 +7,6 @@ import org.simple.clinic.mobius.first
 class OverdueSearchInit : Init<OverdueSearchModel, OverdueSearchEffect> {
 
   override fun init(model: OverdueSearchModel): First<OverdueSearchModel, OverdueSearchEffect> {
-    return first(model, LoadOverdueSearchHistory)
+    return first(model, LoadOverdueSearchHistory, ShowKeyboard)
```

Comment:
This will not show keyboard when we are going back from patient summary screen. We instead call the `overdueSearchQueryEditText.showKeyboard` directly in `onViewCreated`, WDYT?